docs/static/arcsight-module.asciidoc

```diff
@@ -17,6 +17,22 @@ With a single command, the module taps directly into the ArcSight Smart Connecto
 parses and indexes the security events into Elasticsearch, and installs a suite of Kibana dashboards 
 to get you exploring your data immediately. 
 
+[[arcsight-prereqs]]
+==== Prerequisites
+
+These instructions assume that:
+
+* Logstash, Elasticsearch, and
+Kibana are already installed. The products you need are
+https://www.elastic.co/downloads[available to download] and easy to install. The
+Elastic Stack 5.6 or later is required for this module.  
+If you are using the Elastic Stack 6.2 and earlier, please see the instructions for those versions.
```

Comment:
Can we link them to the 6.2 instructions ?

I assume that a 6.3+ will be added here later ?

Reply:
Yes. And yes.